dcrjson/chainsvrresults.go

```diff
@@ -89,16 +89,29 @@ type GetBlockVerboseResult struct {
 	NextHash      string        `json:"nextblockhash,omitempty"`
 }
 
+// AgendaInfo provides an overview of an agenda in a consensus deployment.
+type AgendaInfo struct {
+	Version    uint32 `json:"version"`
```

Comment:
noted, updated. I initially included it for ease of use with `getvoteinfo`, that requires the development id and the version. `getblockchaininfo` would be requested first and with that the caller would have all the info needed to take a closer look at each vote since the response has the deployment id and the version.